apps/docs/content/guides/platform/advanced-log-filtering.mdx

```diff
@@ -0,0 +1,193 @@
+```
+
+### Ignore case sensitivity:
+
+`(?i)` ignores capitalization for all proceeding characters
+
+```sql
+-- find all event_messages with the word "connection"
+regexp_contains(event_message, '(?i)COnnecTion')
+```
+
+### Wildcards:
+
+`.` can represent any string of characters
+
+```sql
+-- find event_messages like "hello<anything>world"
+regexp_contains(event_message, 'hello.world')
+```
+
+### Alphanumeric ranges:
+
+`[1-9a-zA-Z]` finds any strings with only numbers and letters
+
+```sql
+-- find event_messages that contain a number between 1 and 5 (inclusive)
+regexp_contains(event_message, '[1-5]')
+```
+
+### Repeated values:
+
+`x*` zero or more x
+`x+` one or more x
+`x?` zero or one x
+`x{4,}` four or more x
+`x{3}` exactly 3 x
+
+```sql
+-- find event_messages that contains any sequence of 3 digits
+regexp_contains(event_message, '[0-9]{3}')
+```
+
+### Escaping reserved characters:
+
+`\.` interpreted as period `.` instead of as a wildcard
+
+```sql
+-- escapes .
+regexp_contains(event_message, 'hello world\.')
+```
+
+### `or` statements:
+
+`x|y` any string with `x` or `y` present
+
+```sql
+-- find event_messages that have the word 'started' followed by either the word "host" or "authenticated"
+regexp_contains(event_message, 'started host|authenticated')
+```
+
+### and/or/not statements in SQL:
+
+`and`, `or`, and `not` are all native terms in SQL and can be used in conjunction with regular expressions to filter results
+
+```sql
+select
+  cast(timestamp as datetime) as timestamp,
+  event_message,
+  metadata
+from postgres_logs
+where
+  (regexp_contains(event_message, 'connection') and regexp_contains(event_message, 'host'))
+  or not regexp_contains(event_message, 'received');
+```
+
+### Filtering and unnesting example
+
+**Filter for Postgres**
+
+```sql
+select
+  cast(postgres_logs.timestamp as datetime) as timestamp,
+  parsed.error_severity,
+  parsed.user_name,
+  event_message
+from
+  postgres_logs
+  cross join unnest(metadata) as metadata
+  cross join unnest(metadata.parsed) as parsed
+where regexp_contains(parsed.error_severity, 'ERROR|FATAL|PANIC')
+order by timestamp desc
+limit 100;
+```
+
+## Limitations
+
+### Log tables cannot be joined together
+
+Each product table operates independently without any relationships to other log tables. This means they cannot be joined together.
+
+### The `with` keyword and subqueries are not supported
+
+The query editor parses logs for optimization. The parser does not yet support `with` and subquery statements.
+
+### The `ilike` and `similar to` keywords are not supported
+
+Although `like` and other comparison operators can be used, `ilike` and `similar to` are incompatible with BigQuery's variant of SQL. `regexp_contains` can be used as an alternative.
+
+### The wildcard operator `*` to select columns is not supported
+
+The log parser is not able to parse the `*` operator for column selection. Instead, you can access all fields from the `metadata` column:
```

Comment:
This is purposefully restricted as a performance and cost optimization to cut down data scanned and processed.